Pr0gramm/Pr0gramm/Detail/DetailViewController.swift

```diff
@@ -134,6 +134,51 @@ class DetailViewController: ScrollingContentViewController, Storyboarded {
             setupVideo(for: item)
         }
     }
+    
+    func upvotePost() {
+        self.viewModel.vote(.up)
+        self.didFinishUpvote()
+    }
+    
+    func downvotePost() {
+        self.viewModel.vote(.down)
+        self.didFinishDownvote()
+    }
+    
+    func favouritePost() {
```

Comment:
American English bitte :D

Reply:
Ok, ich wohne halt in Europa, daher nehm ich immer British English

Reply:
aber wenn schon favorite drunter :D

Reply:
Erledigt.